scripts/common/preflights.sh

```diff
@@ -657,3 +658,18 @@ function bail_if_unsupported_openebs_to_rook_version() {
     fi
 }
 
+# bail_when_requires_minio_with_openebs will bail if OpenEBS and Minio not when it is required
+function bail_when_requires_minio_with_openebs() {
+    if [ -z "$MINIO_VERSION" ] && [ -n "$OPENEBS_VERSION" ]; then
+        if [ -n "$KOTSADM_VERSION" ] && [[ -z "$KOTSADM_DISABLE_S3" || "$KOTSADM_DISABLE_S3" != "1" ]];then
+             bail "OpenEBS and kOTS with s3 enabled requires MinIO"
+        fi
+        if [ -n "$REGISTRY_VERSION" ]  && [ "$HA_CLUSTER" = "1" ];then
+             bail "OpenEBS and Registry for HA installs requires MinIO. Please, ensure that your installer also provides MinIO"
+        fi
```

Comment:
we back registry with a pvc if there is no object store

```suggestion
```

Reply:
I did not understand this one. What do you mean with `we back registry with a pvc if there is no object store`?
Are you saying that we do not need to add the check because it will fail in another point?

Reply:
It is not multi node velero and registry also requires?